packages/plugin-credentials/src/scope.js

```diff
@@ -0,0 +1,39 @@
+/**!
+ *
+ * Copyright (c) 2015-2016 Cisco Systems, Inc. See LICENSE file.
+ * @private
+ */
+
+/**
+ * sorts a list of scopes
+ * @param {string} scope
+ * @returns {string}
+ */
+export function sortScope(scope) {
+  if (!scope) {
+    return ``;
+  }
+
+  return scope
+    .split(` `)
+    .sort()
+    .join(` `);
+}
+
+/**
+ * sorts a list of scopes and filters the specified scope
+ * @param {string} toFilter
+ * @param {string} scope
+ * @returns {string}
+ */
+export function filterScope(toFilter, scope) {
+  if (!scope) {
+    return ``;
+  }
+
+  return scope
+    .split(` `)
+    .filter((item) => item !== `spark:kms`)
```

Comment:
Should `spark:kms` be passed in as a parameter here? Is there ever another scope we would want to filter